xml/sw_registers.xml

```diff
@@ -38,6 +39,28 @@
         \end{tabular}
         \end{table}
 
+        \begin{table}
+        \centering
+        \caption{Expected Landing Pad Encoding}
+        \label{tab:elpmode}
+        \begin{tabular}{|c|l|}
+        \hline
+        elp &amp; Name \\
+        \hline
+        0 &amp; {\tt NO\_LP\_EXPECTED} \\
+        \hline
+        1 &amp; {\tt LP\_EXPECTED} \\
+        \hline
+        \end{tabular}
+        \end{table}
```

Comment:
I think I prefer these values inlined in `dcsr` and `priv` using the XML value tags. `v` is a bit different since a single table contains `v`, `prv`, and what they mean depending on whether H is supported.

Reply:
Could also just reference Zicfilp for the syntax  and semantics of `ELP`

Reply:
If it's literally just a gdb user interface thing then we might as well split out `$elp` from `$priv` but I wasn't sure how much of the stack splitting the two would touch and whether the new register might cause problems with debug middleware that doesn't otherwise need changes.